en/docs/learn/configuring-recaptcha-for-password-recovery.md

```diff
@@ -47,8 +47,8 @@ reCaptcha for a specific tenant.
     ![account-recovery](../assets/img/using-wso2-identity-server/account-recovery.png) 
 
 7.  You have now successfully configured reCaptcha for the password
-    recovery flow. Start WSO2 Identity Server and log into the end user
-    [user portal](https://localhost:9443/user-portal).
+    recovery flow. Start WSO2 Identity Server and log into the 
+    [My Account portal](https://localhost:9443/myaccount/).
```

Comment:
Do we need a `/` at end of the URL. check other places as well.

Reply:
The trailing slash will not make any difference or cause any problem, therefore It would be fine to keep this as it is. WDYT :) ?

Reply:
Yeah, but looks more clean and nicer without it.